civihr_default_theme/scss/pages/_my-leave.scss

```diff
@@ -0,0 +1,10 @@
+.block-civihr-leave-absences {
```

Comment:
I don't see this being used anywhere, probably because now the template resides somewhere else than your original block?